### PR TITLE
docs: fixed multiple mistakes in the search docs

### PR DESCRIPTION
**Pull request by BenjaminPabst**

### CODE REVIEW

File: docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc [hunk 1]
```diff
@@ -12,12 +12,27 @@ _Adapt in File:_ `+<feature>-bff.yaml+`
 
 * The `+<FeatureName>SearchRequest+` data structure in the BFF yaml must be adapted. 
 
-** All the data that should be displayed in the request must be in that data structure either on top-level or in sub data structures. 
+** All the data that should be used in the request must be in that data structure either on top-level or in sub data structures. 
 
----
+[source, yml]
+----
+<Feature>SearchRequest:
```
Comment (KimFFVII) [inline docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc:19]: < FeatureName >SearchRequest or Search< FeatureName >Request?

File: docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc [hunk 2]
```diff
@@ -12,12 +12,27 @@ _Adapt in File:_ `+<feature>-bff.yaml+`
...
+      type: object
+      properties:
+        limit:
```
Comment (KimFFVII) [inline docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc:22]: is the property `format: int32/64` required here?

File: docs/modules/general/pages/getting_started/search/updatePermissions.adoc [hunk 3]
```diff
@@ -12,7 +12,7 @@ permission:
     enabled: true
     spec:
         permissions:
-            EXAMPLE:
+            FEATURE:
                 CREATE: Create example
```
Comment (KimFFVII) [inline docs/modules/general/pages/getting_started/search/updatePermissions.adoc:16]: CREATE: Create ~~example~~ **feature**?

File: docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc (hunk 1 above)
Comment (BenjaminPabst) [reply docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc:19]: The generated code matches to FeatureSearchRequest that's why I changed it

File: docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc (hunk 2 above)
Comment (BenjaminPabst) [reply docs/modules/general/pages/getting_started/search/configureSearchCriteria.adoc:22]: dont think so

**Pull request merged**